client/common/apollo.tsx

```diff
@@ -30,12 +30,14 @@ function create(initialState: any, headers: IncomingHttpHeaders) {
     headers,
     fetch,
     onError: ({ graphQLErrors, networkError }) => {
-      if (networkError) {
+      if (networkError && process.browser) {
```

Comment:
This error handler could be run server-side, in which case it throws even more errors if trying to render this notification.